R/parse.R

```diff
@@ -25,7 +25,12 @@ parse_text <- function(text, registry = default_tags(), global_options = list())
 }
 
 parse_blocks <- function(file, env, registry, global_options = list()) {
-  parsed <- parse(file = file, keep.source = TRUE)
+
+  lines <- readLines(file, warn = FALSE, encoding = "UTF-8")
```

Comment:
As below, it would be simpler to do 

```R
con <- file(file, encoding = "UTF-8")
on.exit(close(con))
parsed <- parse(con, keep.source = TRUE)
```